src/inspect_ai/_view/www/App.mjs

```diff
@@ -145,6 +155,8 @@ export function App() {
       console.log(e);
       setStatus({ loading: false, error: e });
     }
+
+    mainAppRef.current.focus();
```

Comment:
Is this a new behavior? (focusing at initial load). Are we sure we want to do this?

Reply:
Yes this is, and I think we don't want it (I put it in to make shortcuts responsive immediately, but as you likely are thinking, this steals focus when doing something like editing a document). I'll revert.